.github/ISSUE_TEMPLATE/alpha_feature.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Alpha Feature Requirements
+about: Requirements for promoting a feature to alpha
+---
+
+This page lists the requirements for promoting a feature to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Requires approved RFC in order to merge. 
+
+**Config**
+
+- [ ] Requires explicit user action to enable (e.g. a config field, config resource, or installation action). 
+
+**Docs**
+
+- [ ] Reference docs are published to istio.io. 
+- [ ] Basic feature docs are published on istio.io describing what the feature does, how to use it, and any caveats. 
+- [ ] Whenever possible, documentation has automated istio.io tests written. 
+- [ ] A reference to the design doc/issue is published on istio.io. 
+- [ ] Release notes entries added as appropriate
+- [ ] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [ ] Test coverage is >= 70%
+- [ ] Integration tests cover core use cases with the feature enabled. 
+- [ ] When disabled, the feature does not affect system stability or performance. 
```

Comment:
This is maybe too nebulous, how will they verify this? Can we make this more explicit as to the actual action they need to take here?

Reply:
Good question. This is something inherited from the previous requirements for promotion of features. 

I've discussed with the performance team checking the performance of an Istio release through automated testing on a weekly basis. Is it good enough to analyze the performance on a weekly basis with the default configuration and assuming the performance hasn't dropped very much, call it good?